lib/Cleantalk/ApbctWP/Deactivator.php

```diff
@@ -30,12 +30,10 @@ public static function deactivation($network_wide)
                 self::deleteTables($wpdb->get_blog_prefix($blog));
                 delete_option('cleantalk_cron'); // Deleting cron entries
 
-                if ( $apbct->settings['misc__complete_deactivation'] ) {
-                    self::deleteAllOptions();
-                    self::deleteMeta();
-                    self::deleteAllOptionsInNetwork();
-                    self::deleteSFWUpdateFolder();
-                }
+                self::deleteAllOptions();
```

Comment:
This will delete the whole WPMS options on plugin deactivation, independent of if user want to delete the plugin or just deactivate. Is this we really want to do?

Reply:
Yes, we will work on it.